scripts/build/reload-packages

```diff
@@ -44,7 +44,7 @@ echo -e "Loading packages into DB ${grey}($LOG_DB)${reset}"
 "${LOCALEXEC_EXE}" load-packages > $LOG_DB 2>&1
 
 
-if [[ "$TEST" != "true" ]]; then
+if [[ "$TEST" == "true" ]]; then
```

Comment:
won't merge - this is just my local trick so I don't have to wait for the `dark-packages` seeding to occur, because it's annoyingly slow.